waiter/config-full.edn

```diff
@@ -50,8 +50,14 @@
                   :min-routers 3
 
                   ;; Use a descriptive name for your cluster;
-                  ;; it's best to keep this unique across clusters:
-                  :name "my-waiter-cluster"}
+                  ;; this name is also used in the token metadata as the root.
```

Comment:
We introduce the word "root" here, but there's no context for what the root is - I think it would be nice to explain what the implications of the root are